### PR TITLE
Ry/game invites

### PR DESCRIPTION
**Pull request by Ryno95**
Basic entry points for /game-invites all using the abstract class and nothing from the db is linked, tried to keep it as simple as possible.

Files created == nest g resource games.

Added a unit test file and an e2e test file. With the db connection and a simple test in both cases. Also added unit test for the DTO. Very similar to currentGames tests, but I don't trust the Dto's yet ehe!

Also removed the update controller and UpdateDto as I imagine it'll be something const until removed or accepted.

#38

### CODE REVIEW

**Review by vincentvis — APPROVED** [review]
it indeed looks a lot like the other PR we've just looked at :)

File: backend/src/pong/game_invite/entities/game-invite.entity.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Check, Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity('game-invites')
```
Comment (livlaming) [inline backend/src/pong/game_invite/entities/game-invite.entity.ts:3]: minor detail but for the constancy I thought entity names in snake case?

(copy pasta from slack)
case:
Class names = PascalCase
File names = kebab-case  (Doet nest ook bij het generaten van resources)
Column names + Table names =  snake_case (postgres bewaard alles in lower case) (edited)

File: backend/src/pong/game_invite/game-invite.controller.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	@Get()
+	findAll() {
+		return this.gameInvitesService.findAll();
+	}
+
+	@Get(':id')
+	findOne(@Param('id') id: number) {
+		return this.gameInvitesService.findOne({ where: { id } });
+	}
+
```
Comment (livlaming) [inline backend/src/pong/game_invite/game-invite.controller.ts:31]: probably on purpose, but is it correct that there is no update functionality? 
as in we can't update game-invite? doe sort of makes sense(?)

File: backend/src/pong/game_invite/entities/game-invite.entity.ts (hunk 1 above)
Comment (Ryno95) [reply backend/src/pong/game_invite/entities/game-invite.entity.ts:3]: Yes!

File: backend/src/pong/game_invite/game-invite.controller.ts (hunk 2 above)
Comment (Ryno95) [reply backend/src/pong/game_invite/game-invite.controller.ts:31]: Exactly, there isn't any info to update as the users stay constant, until the invite is removed :)

**Pull request merged**